### PR TITLE
fix(deps): update stackit sdk modules

### PR DESCRIPTION
**Pull request by stackit-pipeline**
This PR contains the following updates:

| Package | Type | Update | Change |
|---|---|---|---|
| [github.com/stackitcloud/stackit-sdk-go/core](https://togithub.com/stackitcloud/stackit-sdk-go) | require | minor | `v0.10.1` -> `v0.12.0` |
| [github.com/stackitcloud/stackit-sdk-go/services/argus](https://togithub.com/stackitcloud/stackit-sdk-go) | require | minor | `v0.9.5` -> `v0.10.0` |
| [github.com/stackitcloud/stackit-sdk-go/services/dns](https://togithub.com/stackitcloud/stackit-sdk-go) | require | minor | `v0.8.4` -> `v0.9.0` |
| [github.com/stackitcloud/stackit-sdk-go/services/loadbalancer](https://togithub.com/stackitcloud/stackit-sdk-go) | require | minor | `v0.10.0` -> `v0.11.0` |
| [github.com/stackitcloud/stackit-sdk-go/services/logme](https://togithub.com/stackitcloud/stackit-sdk-go) | require | minor | `v0.10.1` -> `v0.13.0` |
| [github.com/stackitcloud/stackit-sdk-go/services/mariadb](https://togithub.com/stackitcloud/stackit-sdk-go) | require | minor | `v0.10.1` -> `v0.13.0` |
| [github.com/stackitcloud/stackit-sdk-go/services/mongodbflex](https://togithub.com/stackitcloud/stackit-sdk-go) | require | minor | `v0.11.1` -> `v0.12.0` |
| [github.com/stackitcloud/stackit-sdk-go/services/objectstorage](https://togithub.com/stackitcloud/stackit-sdk-go) | require | minor | `v0.8.6` -> `v0.9.0` |
| [github.com/stackitcloud/stackit-sdk-go/services/opensearch](https://togithub.com/stackitcloud/stackit-sdk-go) | require | minor | `v0.10.1` -> `v0.13.0` |
| [github.com/stackitcloud/stackit-sdk-go/services/postgresflex](https://togithub.com/stackitcloud/stackit-sdk-go) | require | minor | `v0.12.0` -> `v0.13.0` |
| [github.com/stackitcloud/stackit-sdk-go/services/rabbitmq](https://togithub.com/stackitcloud/stackit-sdk-go) | require | minor | `v0.10.0` -> `v0.13.0` |
| [github.com/stackitcloud/stackit-sdk-go/services/redis](https://togithub.com/stackitcloud/stackit-sdk-go) | require | minor | `v0.10.1` -> `v0.13.0` |
| [github.com/stackitcloud/stackit-sdk-go/services/resourcemanager](https://togithub.com/stackitcloud/stackit-sdk-go) | require | minor | `v0.7.7` -> `v0.8.0` |
| [github.com/stackitcloud/stackit-sdk-go/services/secretsmanager](https://togithub.com/stackitcloud/stackit-sdk-go) | require | minor | `v0.6.0` -> `v0.7.0` |
| [github.com/stackitcloud/stackit-sdk-go/services/ske](https://togithub.com/stackitcloud/stackit-sdk-go) | require | minor | `v0.11.0` -> `v0.12.0` |

---

### Configuration

📅 **Schedule**: Branch creation - At any time (no schedule defined), Automerge - At any time (no schedule defined).

🚦 **Automerge**: Disabled by config. Please merge this manually once you are satisfied.

♻ **Rebasing**: Whenever PR becomes conflicted, or you tick the rebase/retry checkbox.

👻 **Immortal**: This PR will be recreated if closed unmerged. Get [config help](https://togithub.com/renovatebot/renovate/discussions) if that's undesired.

---

 - [ ] If you want to rebase/retry this PR, check this box

---

This PR has been generated by [Renovate Bot](https://togithub.com/renovatebot/renovate).

### CODE REVIEW

File: stackit/internal/services/opensearch/credential/resource.go [hunk 1]
```diff
@@ -157,12 +155,6 @@ func (r *credentialResource) Schema(_ context.Context, _ resource.SchemaRequest,
 				ElementType: types.StringType,
```
Comment (vicentepinto98) [inline stackit/internal/services/opensearch/credential/resource.go:155]: Opensearch credentials has a new string field `Scheme`, can you add it?

File: stackit/internal/services/rabbitmq/credential/resource.go [hunk 2]
```diff
@@ -37,7 +37,6 @@ type Model struct {
 	Host         types.String `tfsdk:"host"`
 	Hosts        types.List   `tfsdk:"hosts"`
 	HttpAPIURI   types.String `tfsdk:"http_api_uri"`
-	Name         types.String `tfsdk:"name"`
```
Comment (vicentepinto98) [inline stackit/internal/services/rabbitmq/credential/resource.go:40]: RabbitMQ credentials has new fields:
- `HttpApiUris`, list of strings (similar implementation as `hosts`)
- `Management`, string
- `Uris`, list of strings (similar implementation as `hosts`)

Can you add them?

File: stackit/internal/services/redis/credential/resource.go [hunk 3]
```diff
@@ -36,8 +36,6 @@ type Model struct {
 	ProjectId    types.String `tfsdk:"project_id"`
 	Host         types.String `tfsdk:"host"`
 	Hosts        types.List   `tfsdk:"hosts"`
-	HttpAPIURI   types.String `tfsdk:"http_api_uri"`
```
Comment (vicentepinto98) [inline stackit/internal/services/redis/credential/resource.go:39]: Redis credentials has a new string field `LoadBalancedHost`, can you add it?

File: stackit/internal/services/rabbitmq/credential/resource.go (hunk 2 above)
Comment (GokceGK) [reply stackit/internal/services/rabbitmq/credential/resource.go:40]: added all new fields

File: stackit/internal/services/opensearch/credential/resource.go (hunk 1 above)
Comment (GokceGK) [reply stackit/internal/services/opensearch/credential/resource.go:156]: added new field

**Comment by vicentepinto98** [comment]
Are these fields also not being read in the datasources? Nonetheless, the new fields should also be added for the datasources

**Comment by vicentepinto98** [comment]
Please also check and adapt the acceptance tests if necessary

**Comment by vicentepinto98** [comment]
Did the acceptance tests pass?

**Comment by GokceGK** [comment]
> Did the acceptance tests pass?

yes the updated acceptance tests pass.

**Comment by stackit-pipeline** [comment]
### Edited/Blocked Notification

Renovate will not automatically rebase this PR, because it does not recognize the last commit author and assumes somebody else may have edited the PR.

You can manually request rebase by checking the rebase/retry box above.

 ⚠ **Warning**: custom changes will be lost.

**Review by vicentepinto98 — APPROVED** [review]

**Pull request merged**